src/libraries/System.Collections/src/System/Collections/Generic/SortedSet.cs

```diff
@@ -1922,11 +1923,13 @@ internal Enumerator(SortedSet<T> set, bool reverse)
                 Initialize();
             }
 
+            [UnsupportedOSPlatform("browser")]
```

Comment:
This throws everywhere. Why mark it as unsupported for browser only?

Reply:
That's good question, thanks. I'm not sure if there is something like `[UnsupportedOSPlatform("all")]` to mark it for all the possible platforms. So in this case I've done it only for the related part.